python/mxnet/gluon/contrib/estimator/event_handler.py

```diff
@@ -130,13 +130,15 @@ class MetricHandler(EpochBegin, BatchEnd):
     ----------
     train_metrics : List of EvalMetrics
         Training metrics to be updated at batch end.
+    priority : scalar
+        Priority level of the MetricHandler
```

Comment:
Can you explain that the priority is used for ascending sorting of the handlers?

Reply:
I have fixed these documentation issues.